tests/test_sqlalchemy_data_layer.py

```diff
@@ -1649,3 +1649,31 @@ class PersonComputersOwnedRelationship(person_computers):
         '/persons/{}/relationships/computers-owned'.format(person.person_id),
         content_type='application/vnd.api+json')
     assert response.status_code == 200, response.json['errors']
+
+
+def test_post_list_nested_field(app, api, client, session, post_schema, post_model):
+
```

Comment:
Can you add a comment here explaining the test? Something like "ensure a schema containing a nested field is correctly serialized and deserialized".